Makefile

```diff
@@ -65,3 +65,13 @@ deb: build
 		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
 		--maintainer "$(MAINTAINER)" \
 		test/config/ sa/_db data/ $(OBJECTS) bin/ct-test-srv
+
+tar: build
+	fpm -f -s dir -t tar --name "boulder" \
```

Comment:
It didn't occur to me to use fpm to make the tarball! This is a great idea.

It's surprising to me that all of these same flags (e.g. --vendor, --iteration) exist/work for the `-t tar` mode, but it sure looks like `fpm` is happy to accept them! I wonder if/how they actually are reflected in the final archive...

Reply:
All of the following are reflected in the result:

```
--name
--prefix
--iteration
```

I believe the rest of the flags and args can be removed.

Filename:
```
boulder-1.0.0-455cdade.amd64.tar.gz
```

```
$ tree opt
opt
└── boulder
    ├── bin
    │   ├── admin-revoker
    │   ├── akamai-purger
    │   ├── bad-key-revoker
    │   ├── boulder
    │   ├── boulder-ca
    │   ├── boulder-observer
    │   ├── boulder-publisher
    │   ├── boulder-ra
    │   ├── boulder-sa
    │   ├── boulder-va
    │   ├── boulder-wfe2
    │   ├── caa-log-checker
    │   ├── ceremony
    │   ├── cert-checker
    │   ├── contact-auditor
    │   ├── crl-storer
    │   ├── crl-updater
    │   ├── expiration-mailer
    │   ├── id-exporter
    │   ├── log-validator
    │   ├── nonce-service
    │   ├── notify-mailer
    │   ├── ocsp-responder
    │   ├── ocsp-updater
    │   ├── orphan-finder
    │   ├── reversed-hostname-checker
    │   └── rocsp-tool
    ├── data
    │   ├── production-email.template
    │   └── staging-email.template
    ├── sa
    │   └── _db
    │       ├── dbconf.yml
    │       └── migrations
    │           ├── 20210223140000_CombinedSchema.sql
    │           ├── 20210308140000_SimplePartitioning.sql
    │           └── 20220328100000_Incidents.sql
    └── test
        └── config
            ├── admin-revoker.json
            ├── akamai-purger.json
            ├── bad-key-revoker.json
            ├── ca-a.json
            ├── ca-b.json
            ├── cert-checker.json
            ├── contact-auditor.json
            ├── contact-exporter.json
            ├── crl-storer.json
            ├── crl-updater.json
            ├── ecdsaAllowList.yml
            ├── expiration-mailer.gotmpl
            ├── expiration-mailer.json
            ├── health-checker.json
            ├── log-validator.json
            ├── nonce.json
            ├── notify-mailer.json
            ├── observer.yml
            ├── ocsp-responder.json
            ├── ocsp-updater.json
            ├── orphan-finder.json
            ├── publisher.json
            ├── ra.json
            ├── rocsp-tool.json
            ├── sa.json
            ├── va-remote-a.json
            ├── va-remote-b.json
            ├── va.json
            └── wfe2.json

8 directories, 62 files
```

Reply:
Are even those ones actually used? It looks to me like the filename is determined by the `--package` flag, rather than constructed from the values of those separate flags.

But I'm not sure what other kinds of metadata .tar files are capable of containing, and whether any of these other flags get baked into the tar somewhere other than the filename and file list.

Reply:
> edit: Empirical testing shows that --name boulder is necessary for fpm to be happy. The --prefix and --package flags are used to control the filename and the filepaths within the tar. Other than that, I can't observe these affecting anything... (but maybe I'm looking in the wrong place).

I just came here to post this.